src/api/api.test.ts

```diff
@@ -0,0 +1,36 @@
+import { vi } from 'vitest';
+import { getCatFetch } from './api';
+
+it('calls the right route', async () => {
+  const mockCat = { id: 1, name: 'Бенедикт', breed: 'Абиссинская', price: 2000 };
+  const Respons = Promise.resolve(mockCat);
+  const mockFetch = Promise.resolve({
+    json: () => Respons,
+  });
+  global.fetch = vi.fn().mockImplementation(() => mockFetch);
```

Comment:
Firstly, it is necessary to revert default implementation of fetch function.
Secondly, spyOn is not needed if you change fetch property in global object